src/registrationTracking.js

```diff
@@ -0,0 +1,94 @@
+import{pushInDataLayer} from './modules/pushInDataLayer';
+import{checkEmptyField} from './modules/checkEmptyField';
+
+window.addEventListener('DOMContentLoaded', () => {
+    const regForm = document.getElementById('regForm');
+    const errorsArray = Array.from(document.querySelectorAll('.field-validation-error'));
+    let errorMessages = [];
```

Comment:
For consistency I'd consider making `errorMessages` a `const` too (like `errorsArray`)

Reply:
Done 👍